### PR TITLE
Adds Antag Tokens to the Beecoin Shop

### PR DESCRIPTION
**Pull request by AsciiSquid**
## About The Pull Request
Adds antag tokens to the beecoin shop, much to Kev's displeasure. antag_token
The antag token is available for 10,000 beecoins, as it is a pretty powerful thing to buy.

## Why It's Good For The Game
It isn't. But I know a lot of people want to buy antag tokens from the beecoin shop so I'd figured I would add it. I feel the price is balanced for what it is.

## Changelog
:cl:
add: Added Antag Token to the Beecoin shop.
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Based

**Comment by despong** [comment]
wait for people to stock up on antag coins and there be like 6 back to back nuclear emergency rounds

**Comment by Arrfur** [comment]
Can I propose some conditions? I'm gonna do it anyway:
-Tokens will only add you to the current round's antag. No choosing
-Team antag rounds (Nukies, ayys, the likes) wont be allowed
-Antag rounds balanced around 1 player wont be allowed (basically wiz)
-Antag tokens will need to be used in the pre-game lobby or shortly after the round beings (say 2 mins?)
-Using a token on a conversion round will make you a leader
-YOU WONT BE TOLD WHAT ANTAG IT IS UNLESS YOU ACTUALLY USE IT, NO SNEAKY BEAKY HERE SIR NO NOT TODAY.

Basically the gist of this is that tokens wont be able to be used for big shid, ,mostly just an extra ling or tot or culti or whatever. 
toolbox already has a system for players to redeem tokens themselves, maybe we could """"""""""borrow"""""""""
(if we do be careful bc their system gives you a token every 4 hrs of play))

File: code/modules/client/loadout/loadout_general.dm [hunk 1]
```diff
@@ -2,3 +2,8 @@
 	display_name = "d20"
 	path = /obj/item/dice/d20
 	cost = 300
+
+/datum/gear/antagtoken
+	display_name = "Antag Token"
+	path = /obj/item/coin/antagtoken
+	cost = 10000
```
Comment (ike709) [inline code/modules/client/loadout/loadout_general.dm:9]: Unironically make it 100000

**Comment by VictorVow** [comment]
Instead of Arrfur's suggestions I'd say just use the existing antag token rules, but also allow tokens to be used on conversion rounds.

If it's either cult then you simply become another member. But if it's Gang War then you become the Gang Leader of a new gang, if it's Revolution then you become a HeadRev (only if there isn't three yet) and if it's Jungle Fever you become a Monkey Leader.

**Comment by qwertyquerty** [comment]
How cute, they're all so clueless and ignorant

**Comment by ike709** [comment]
@qwertyquerty It would've been funnier if it was 100,000.

**Comment by despong** [comment]
make it 10 quadrillion for a real one
